### ISSUE TITLE
Artifact ID "workflow-server" Re-Used

### ISSUE THREAD

**Issue by stefanpieper** [CoreMedia/coremedia-globallink-connect-integration#16]
# Description

In release 2004.1 (and maybe before), the root module for the workflow server's extension uses artifact ID _workflow-server_ - just as the workflow server's docker module in Blueprint workspace. If a user integrates the GlobalLink extension and expects a 

```
mvn clean install -pl :workflow-server -am ...
```

to operate as before, s/he will be disappointed and start moaning heart-breakingly.

Could you consider to find another name that will not clash so that people don't step into this trap?

# Workaround

Prepend old workflow-server module with proper group ID, i.e.

```
mvn clean install -pl com.coremedia.blueprint.docker:workflow-server -am ...
```

**Comment by acollet** [comment]
oh yes, this is indeed annoying and one is very confused when building it (no error messages, simply not building which would be expected but just a sub set)..

**Comment by mmichaelis** [comment]
Fixed with release for 2007.1.
